include/hicma_constants.h

```diff
@@ -26,7 +26,10 @@
 #define HICMA_STARSH_PROB_GEOSTAT   5
 #define HICMA_STARSH_PROB_EDSIN  6
 #define HICMA_STARSH_PROB_GEOSTAT_POINT   7
-
```

Comment:
can u please remove this one?

Reply:
It is not from my side. This is the original file.